kargo/stages.yaml

```
@@ -102,11 +102,11 @@ spec:
       - uses: git-push
         config:
           path: ./out
-          targetBranch: stage/uat
+          targetBranch: stage/staging
       - uses: argocd-update
         config:
           apps:
-          - name: kargo-simple-uat
+          - name: kargo-simple-staging
```

```suggestion
          - name: guestbook-simple-staging
```
If we want to make it to work using current app configuration, same for evrywhere